core/server/master/src/main/java/alluxio/master/metastore/rocks/RocksStore.java

```diff
@@ -84,10 +90,29 @@ public RocksStore(String name, String dbPath, String checkpointPath,
       DBOptions dbOpts,
       Collection<ColumnFamilyDescriptor> columnFamilyDescriptors,
       List<AtomicReference<ColumnFamilyHandle>> columnHandles) {
+    this(name, dbPath, checkpointPath, null, null, dbOpts, columnFamilyDescriptors, columnHandles);
+  }
+
+  /**
+   * @param name a name to distinguish what store this is
+   * @param dbPath a path for the rocks database
+   * @param checkpointPath a path for taking database checkpoints
+   * @param parallelBackupPath a path for taking database backup in parallel
+   * @param parallelBackupPoolSize the thread pool size for taking database backup in parallel
+   * @param dbOpts the configured RocksDB options
+   * @param columnFamilyDescriptors columns to create within the rocks database
+   * @param columnHandles column handle references to populate
+   */
+  public RocksStore(String name, String dbPath, String checkpointPath, String parallelBackupPath,
+                    Integer parallelBackupPoolSize, DBOptions dbOpts,
+                    Collection<ColumnFamilyDescriptor> columnFamilyDescriptors,
+                    List<AtomicReference<ColumnFamilyHandle>> columnHandles) {
```

Comment:
nit: +4 spaces on line break
```suggestion
  public RocksStore(String name, String dbPath, String checkpointPath, String parallelBackupPath,
      Integer parallelBackupPoolSize, DBOptions dbOpts,
      Collection<ColumnFamilyDescriptor> columnFamilyDescriptors,
      List<AtomicReference<ColumnFamilyHandle>> columnHandles) {
```